src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx

```diff
@@ -17,12 +17,52 @@ const ManageSupportUI = () => {
           </div>
         }
       >
+        <span className={styles.warning}>
+          <strong> Warning! </strong>`Agreement` is a parent camp to the list of
```

Comment:
It should be from the common message file

Reply:
It is a static design implementation for now. will be dynamic once API will be implemented.